scripts/postprocess_sitemap.py

```diff
@@ -0,0 +1,25 @@
+from bs4 import BeautifulSoup
+
+with open('./_build/html/sitemap.xml', 'r') as f:
+    contents = f.read()
+
+soup = BeautifulSoup(contents, 'lxml')
```

Comment:
When I run this locally, the first time I make this call it says:
> It looks like you're parsing an XML document using an HTML parser. If this really is an HTML document (maybe it's XHTML?), you can ignore or filter this warning. If it's XML, you should know that using an XML parser will be more reliable. To parse this document as XML, make sure you have the lxml package installed, and pass the keyword argument `features="xml"` into the BeautifulSoup constructor.

If I pass in `'xml'` instead of `'lxml'` it doesn't give the warning:
```suggestion
soup = BeautifulSoup(contents, 'xml')
```
so it's probably worth making the change.

Reply:
Which command gives that warning? For me,`make html` or `make livehtml` doesn't give me any warning
<img width="949" alt="Screenshot 2023-07-06 at 13 16 41" src="https://github.com/aiven/devportal/assets/3796599/c175b2a3-0e33-4826-8dd2-e341e7b42f30">

But we can update to `xml` since is for `sitemap.xml `and the XML is well-formed and doesn't have any peculiarities that might confuse an XML parser.